packages/cozy-urls/src/cozy-urls.js

```diff
@@ -6,12 +6,17 @@ const ERROR_PROTOCOL_URL_INVALID = `[cozy-urls] Can't find protocol for`
 
 const DOMAIN_REGEX = '(((https?):)?/{2})?(([^.]*.)*[^./:]*)'
 
+let cozyURL
```

Comment:
I think there is a potential issue with this on mobile app. If we use `getCozyURL`, then logout, this variable will still contain the old URL. If we login to another instance, the old URL will remain.